Resources/Textures/Objects/Specific/Robotics/borgmodule.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
```

```suggestion
    "version": 2,
```
А так всё норм по прототипам, как я поглядел